jobs/etcd/spec

```diff
@@ -56,9 +58,29 @@ properties:
     description: Certificate for etcdctl client authentication
   tls.etcdctl.private_key:
     description: Private key for etcdctl client authentication
+  tls.etcdctl-flanneld.certificate:
+    description: Certificate for etcdctl used by flanneld job client authentication
+  tls.etcdctl-flanneld.private_key:
+    description: Private key for etcdctl used by flanneld job client authentication
   tls.peer.ca:
     description: CA for peer authentication
   tls.peer.certificate:
     description: Certificate for peer authentication
   tls.peer.private_key:
     description: Private key for peer authentication
+  users:
```

Comment:
This list should probably let you choose the API version to create the user under - `flanneld` for example needs to be created in `v2`, but root probably needs to exist on both `v2` and `v3`.

e.g.

```
- name: root
  password: ((etcd_user_root_password))
  versions: [ "v2", "v3" ]
```

Reply:
Well the problem is the etcdctl command for v2 and v3 have big difference. But yeah let me try then.